index.js

```diff
@@ -53,6 +53,7 @@ class CaverExtKAS extends Caver {
             feeDelegatedAccountUpdate: this.transaction.feeDelegatedAccountUpdate,
             createWithAccountKeyPublic: this.account.createWithAccountKeyPublic,
         }
+        this.kas.wallet.keyring = this.wallet.keyring
```

Comment:
Is this essential? If not, i think it is better to guide to user to use `caver.keyringContainer.keyring`.

Reply:
Oh my bad. i understand why this is needed :)